corehq/apps/locations/bulk_management.py

```diff
@@ -860,7 +861,14 @@ def append_at_bottom(parent_type):
                 if loc.needs_save:
                     loc_object = loc.db_object
                     loc_object.location_type = types_by_code.get(loc.location_type)
-                    loc_object.parent = loc.new_parent
+                    parent_code = loc.parent_code
+                    if parent_code == ROOT_LOCATION_TYPE:
+                        loc_object.parent = None
```

Comment:
Should not this raise an error instead of just removing the parent?

Reply:
This isn't an error - the user is specifying that the location has no parent.

Reply:
Aha.. so we assign the `parent`, the user just assigns the parent_code. Cool!